### DISCUSSION TITLE
How to check integrity of downloaded binaries?

### DISCUSSION THREAD

**Discussion by jdpiguet** [arendst/Tasmota#18357] [Q&A] [answered]
Hi all,
I would like to check the integrity of the binaries I am downloading from "https://ota.tasmota.com/tasmota/release/".
How can I do it?
Do you have a list of checksums (MD5, SHAx, ...) somewhere?
Regards,
Jacques-D.

**Comment by arendst** [comment, marked as answer]
Nope. No checksums are currently generated.

**Reply by jdpiguet** [reply]
And anyone planned in any future ;-)

**Reply by sfromis** [reply]
Nope
